src/main/java/com/atlassian/bitbucket/jenkins/internal/link/BitbucketMultibranchLinkActionFactory.java

```diff
@@ -0,0 +1,47 @@
+package com.atlassian.bitbucket.jenkins.internal.link;
+
+import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCMRepository;
+import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCMSource;
+import hudson.Extension;
+import hudson.model.Action;
+import jenkins.model.TransientActionFactory;
+import org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+
+@Extension
+public class BitbucketMultibranchLinkActionFactory extends TransientActionFactory<WorkflowMultiBranchProject> {
+
+    @Inject
+    private BitbucketExternalLinkUtils externalLinkUtils;
+
+    public BitbucketMultibranchLinkActionFactory() { }
+
+    @Inject
```

Comment:
double injection annotations again, I think we only need one of these annotations?